date_range/static/src/js/date_range.js

```diff
@@ -43,9 +49,15 @@ odoo.define('date_range.search_filters', function (require) {
 
         date_range_type_operator_selected: function (type_id) {
             this.$value.empty().show();
+//            this._rpc({
+//                model: 'date.range',
+//                method: 'search_read',
+//                fields: ['name', 'date_start', 'date_end'],
+//                context: this.context,
+//                domain: ['type_id', '=', parseInt(type_id, 10)]]
+//                }).done(this.proxy('on_range_type_selected'));
```

Comment:
@mreficent This code must be removed id it's no more needed

Reply:
I added this code expressly to indicate what should be the way to fix the javascript error, this code should replace the next two lines. I put commented instead because it's not correct yet.